### DISCUSSION TITLE
Audio sound

### DISCUSSION THREAD

**Discussion by dinarpay** [baresip/baresip#2683] [Q&A]
I am working on a Linux server via the command line, and there's no sound card on the server. I want to make a call without encountering this message. Can I know the correct commands to operate and a solution for the sound issue?

ALSA lib confmisc.c:767:(parse_card) cannot find card '0'
ALSA lib conf.c:4528:(_snd_config_evaluate) function snd_func_card_driver returned error: No such file or directory
ALSA lib confmisc.c:392:(snd_func_concat) error evaluating strings
ALSA lib conf.c:4528:(_snd_config_evaluate) function snd_func_concat returned error: No such file or directory
ALSA lib confmisc.c:1246:(snd_func_refer) error evaluating name
ALSA lib conf.c:4528:(_snd_config_evaluate) function snd_func_refer returned error: No such file or directory
ALSA lib conf.c:5007:(snd_config_expand) Evaluate error: No such file or directory
ALSA lib pcm.c:2495:(snd_pcm_open_noupdate) Unknown PCM default
alsa: could not open auplay device 'default' (No such file or directory)

**Comment by sreimers** [comment]
You can use the `aubridge` module as dummy source/destination:
```
audio_player            aubridge,default
audio_source            aubridge,default
audio_alert             aubridge,default
```
